### ISSUE TITLE
NTR: Caustic substance ingestion, recent (GI, feeding, vomiting & dehydration)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7892]
New term request Caustic substance ingestion, recent (brumme_220518150036): Ingestion poisoning includes cleaning products, acidic and alkaline, accidental and intentional  UMLS: 0742008

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000223
- label: Recent history of caustic substance ingestion
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: Medical history of a recent ingestion of a caustic substance, i.e., a highly acidic or alkaline chemicals that can cause severe burns to the mouth and digestive tract when swallowed.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7892
- synonyms:
